src/main/java/RecrBad.java

```diff
@@ -0,0 +1,162 @@
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+
+public class RecrBad {
+
+    private static Task[] addEventTask(final int startIndexOfDescription, String line, Task[] tasks){
+        Task[] moreTasks = Arrays.copyOf(tasks, tasks.length + 1);
+        int indexTo = line.lastIndexOf('/');
+        int indexFrom = line.lastIndexOf('/', indexTo - 1); // counts from back
+        if (indexFrom == -1 || indexTo == -1) { //not found
+            System.out.println("Please input BOTH start and end timing parameters by adding '/' followed by the start timing, then ' ', '/' and end timing");
+            return tasks;
+        }
+        String description = line.substring(startIndexOfDescription, indexFrom - 1);
+        String timeRange = " (from: " + line.substring(indexFrom + 1, indexTo) +
+                "to: " + line.substring(indexTo + 1) + ")";
+        moreTasks[tasks.length] = new Event(description, timeRange); //append at last elem
+        System.out.println("Event added!");
+        return moreTasks;
+    }
+    private static Task[] addDeadlineTask(int startIndexOfDescription, String line, Task[] tasks){
+        Task[] moreTasks = Arrays.copyOf(tasks, tasks.length + 1);
+        int indexDeadline = line.lastIndexOf('/');
+        if (indexDeadline == -1) { // no deadline argument
+            System.out.println("Please input a deadline by adding a parameter '/' followed by the deadline");
+            return tasks;
+        }
+        String deadline = "(by: " + line.substring(indexDeadline + 1) + ")";
+        String description = line.substring(startIndexOfDescription, indexDeadline);
+        moreTasks[tasks.length] = new Deadline(description, deadline); //append at last elem
+        System.out.println("Deadline added!");
+        return moreTasks;
+    }
+    private static Task[] addTodoTask(final int startIndexOfDescription, String line, Task[] tasks){
+        Task[] moreTasks = Arrays.copyOf(tasks, tasks.length + 1);
+        String description = line.substring(startIndexOfDescription);
+        moreTasks[moreTasks.length - 1] = new Todo(description); //append at last elem
+        System.out.println("Todo added!");
+        return moreTasks;
+    }
+
+    /**
+     * Adds new Task to Tasks array
```

Comment:
Clear demonstration in javadoc, let me clear know what each method does, which is good practice.